modules/__tests__/dynamicRoutes/CheckoutPage.js

```diff
@@ -0,0 +1,7 @@
+import { Component } from 'react'
```

Comment:
No need for a separate file for each component. Please just put it in the same file like [this](https://github.com/rackt/react-router/blob/e7c6f3d848e55dda11595447928e843d39bed0eb/modules/__tests__/Router-test.js#L12-L22)